runtime/cere-dev/src/lib.rs

```diff
@@ -1317,7 +1317,7 @@ impl pallet_ddc_verification::Config for Runtime {
 	type OffchainIdentifierId = ddc_primitives::crypto::OffchainIdentifierId;
 	type ActivityHasher = BlakeTwo256;
 	const MAJORITY: u8 = 67;
-	const BLOCK_TO_START: u16 = 30; // every 30 blocks
+	const BLOCK_TO_START: u16 = 1; // every block
```

Comment:
You will face with the `WARN tokio-runtime-worker wasm-runtime: Ran out of free WASM instances` issue in case you launch it every block. This issue is the reason of why we had failed connections to RPC/Archive nodes in the past.
I guess before we set this value we need to find the appropriate configuration of the node. 
For now, I suggest to set this value to at least 10 blocks that is 1 minute for a trigger

Reply:
Ok, let's try. Possibly the changes you made in this PR will really prevent this problem